.gitignore

```diff
@@ -4,5 +4,5 @@
 /containers/latest/qemu-*-static
 /containers/register/Dockerfile
 /containers/register/register.sh
-/containers/x86_64_qemu-*/
+/containers/qemu-*/
```

Comment:
As I commented in big picture section, I prefer keeping x86_64 prefix, because we are trying support other host arch now. I like to see that this modification in `.gitignore` is reverted.

Reply:
This dir is `rm -rf`-ed after each loop iteration: https://github.com/meeDamian/qemu-user-static/blob/simplify/build-docker.sh#L61, so frankly speaking a completely static name could be used for it.  As a matter of fact, I think it might be even better, and simpler.